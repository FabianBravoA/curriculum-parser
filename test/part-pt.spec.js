'use strict';


const Helpers = require('./helpers');
const Part = require('../lib/part');


describe('Part', () => {

  it('should parse empty part with meta data [pt_BR]', (done) => {
    Part(Helpers.fixturesBasePath, '')('00-part-pt', (err, data) => {
      expect(err).toBe(null);
      expect(data).toMatchSnapshot();
      done();
    });
  });

});
