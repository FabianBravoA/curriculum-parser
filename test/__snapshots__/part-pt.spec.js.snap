// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Part should parse empty part with meta data [pt_BR] 1`] = `
Object {
  "files": Array [
    "README.md",
  ],
  "log": Array [],
  "result": Object {
    "duration": 30,
    "format": "guided",
    "partDir": "00-part-pt",
    "title": "TÃ­tulo",
    "type": "seminar",
    "unitDir": "",
  },
}
`;
